.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.0'
+          - '1'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
```

Comment:
Might be more useful to test for 64 and 32-bit (to catch the `Int32` vs `Int64` common gotcha's) but only on linux.

Reply:
done, I took the one from MOI but added OSX